snowex_db/point_primary_variable_overrides.yaml

```diff
@@ -224,3 +225,26 @@ WIND_SPEED_10FT:
   map_from:
   - wsms_10ft_avg
   match_on_code: true
+CAMPAIGN:
+  auto_remap: true
+  code: campaign
+  description: Name of campaign site
+  map_from:
+  - location
+  match_on_code: true
+NAME:
+    auto_remap: true
+    code: name
+    description: Name of the measurement
+    map_from:
+    - pit_id
+    - pitid
+    match_on_code: true
+COMMENTS:
```

Comment:
In the campaign overwrites in insitupy for SnowEx, [we kept these separate](https://github.com/jomey/insitupy/blob/4925e524655afaaf9bbab1cf8dbd43c654b4c1ad/insitupy/campaigns/snowex/snowexprimaryvariables.yaml#L30-L43).
Wonder if we can do the same here to stay consistent?